_component_design/key-value-pairs.md

```diff
@@ -1,40 +1,120 @@
 ---
-title: Key Value Pairs
-parent: unknown
+title: Key-value pairs
+parent: Content Areas
 layout: component
 category: Components
 usage: >
-  Key value pairs are used to display data with their corresponding label. Color
-  and alignment are used to enhance readability allowing users to quickly scan
-  for the page for relevant data.
-preview-image: null
+   Key-value pairs display data (the value), that correspond to a label (the key). Color and alignment of key-value pairs allow the user to scan the page quickly for relevant data.
+preview-image: preview-images/key-value.svg
 resource: true
-status: unknown
-last-modified: 2017-08-17
+status: Stable
+last-modified: 2017-11-07
 ---
 
 {% include toc.html %}
 
-Key value pairs are used to display data with their corresponding label. Color
-and alignment are used to enhance readability allowing users to quickly scan
-for the page for relevant data.
+## Introduction
 
-## Best Practices
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### When to use
+
+Use key-value pairs when a key, an identifier for data, needs either a text or visual display of that keys's corresponding value.
+
+### Best practices
+
+- Ensure key values corresponds with the label used for the data.
+- Keep key values limited to one line.
+- Follow the Helix grid guidelines for spacing and placement.
+- Use icons sparingly.
+- Use the vertical key pattern if key value sections are arranged on the page side by side.
+- Consider breaking key-value pairs into sub-groups if there are more than five sets of pairs.
+
+{% endcolumn %}
+</div>
+
+## Specifications and states
 
 <div class="hxRow">
-{% column left:"hxCol-4" %}
--   Labels should correspond with the labels used for data creation and/or
-  editing.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Key-value pairs include the following specifications and states:
+
+### Default layout
+
+Key-value pairs are contained in the body of the page by default. Keys, such as "Ticket ID", are displayed on the left and aligned right. Values, (such as "Summary") are displayed on the right and are aligned left.
```

Comment:
Use markup tags instead of quotes
remove comma pair
... display on the left side and are right-aligned.
Values such as Summary display on the right side and are left-aligned.